manifests/config.pp

```diff
@@ -12,6 +12,16 @@
         changes => "set tftpd_flags '\"-s ${tftp::root}\"'",
       }
     }
+    if $facts['osfamily'] == 'Debian' {
+      file { '/etc/default/tftpd-hpa':
+        ensure  => file,
+        owner   => 'root',
+        group   => 'root',
+        mode    => '0644',
+        content => template('tftp/tftpd-hpa.erb'),
+        notify  => Service[$tftp::service],
```

Comment:
This is redundant because `init.pp` already ensures the config class notifies the service class.

Reply:
removed